packages/components/image/src/image.tsx

```diff
@@ -44,6 +45,19 @@ const Image = forwardRef<"img", ImageProps>((props, ref) => {
     );
   }
 
+  if (isError && fallbackSrc) {
+    const errorImgStyle =
+      "relative z-10 shadow-black/5 shadow-none transition-transform-opacity motion-reduce:transition-none !duration-300 rounded-large";
+
+    const errorImgWrapper = "relative shadow-black/5 shadow-none rounded-large w-max";
+
+    return (
+      <div className={errorImgWrapper}>
+        <img alt={img.props.alt} className={errorImgStyle} src={fallbackSrc as string} />
```

Comment:
`fallbackSrc` should be `string` as indicated in doc. Hence, you may update it in `use-image` isntead of casting.